setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
They are distinct, that's why I suggested IOError deletion.
IOError is when the genuine disk error occurs, or directory doesn't exist.